package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "riemann",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
If the API has changed, do we need to change the docs a bit more? If not, is this still a major bump?

Reply:
I believe that a user could send an Event with keys that are not enforced via strict enumeration (for instance, call such as Event `metric_sint64`) - which, in the case of this new library, would expect metricSint64 (it auto-camel cases underscores in the proto file when converting object to protobuf).  Since this could bubble up to the user callable, it's not strictly an API change, but it would be a regression in behavior.

I guess this could be mentioned in the README regarding breaking change, or updated in the serializer to auto camel case all snake inputs to fully preserve undocumented behavior.

Reply:
Please describe how the API has changed in the description of this pull request.